huggingface/text2text-generation/llama/fingpt-mt-llama3-8b-lora/inference.py

```diff
@@ -0,0 +1,35 @@
+import os
+
+from optimum.rbln import RBLNLlamaForCausalLM
+from transformers import AutoTokenizer
+
+
+def main():
+    model_id = "meta-llama/Meta-Llama-3-8B"
+    lora_id = "FinGPT/fingpt-mt_llama3-8b_lora"
+    prompt = """Instruction: What is the sentiment of this news? Please choose an answer from {negative/neutral/positive}
+Input: FINANCING OF ASPOCOMP 'S GROWTH Aspocomp is aggressively pursuing its growth strategy by increasingly focusing on technologically more demanding HDI printed circuit boards PCBs .
```

Comment:
Can we change the company name here? It would be good to use the name which is not existed in real world.